source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -38,6 +38,7 @@ namespace ExtAuthz {
   COUNTER(denied)                                                                                  \
   COUNTER(error)                                                                                   \
   COUNTER(timeout)                                                                                 \
+  COUNTER(disabled)                                                                                \
```

Comment:
This state should be documented wherever the other stats are documented.

Reply:
updated in both HTTP and TCP ext_authz docs.